electron-app/src/background.ts

```diff
@@ -81,27 +82,112 @@ function createWindow() {
-          { role: 'paste' },
-          { role: 'pasteAndMatchStyle' },
-          { role: 'delete' },
-          { role: 'selectAll' }
+  const defaultMenuTemplate = [
+    // On a mac we want to show a "Rotki" menu item in the
+    ...(isMac
+      ? [
+          {
+            label: app.name,
+            submenu: [
+              { role: 'hide' },
+              { role: 'hideOthers' },
+              { role: 'unhide' },
+              { type: 'separator' },
+              { role: 'quit' }
+            ]
+          }
         ]
-      }
-    ];
-    Menu.setApplicationMenu(Menu.buildFromTemplate(template));
-  }
+      : []),
+    {
+      label: 'File',
+      submenu: [isMac ? { role: 'close' } : { role: 'quit' }]
+    },
+
+    {
+      label: '&Edit',
+      submenu: [
+        { role: 'undo' },
+        { role: 'redo' },
+        { type: 'separator' },
+        { role: 'cut' },
+        { role: 'copy' },
+        { role: 'paste' },
+        // Macs have special copy/paste and speech functionality
+        ...(isMac
+          ? [
+              { role: 'pasteAndMatchStyle' },
+              { role: 'delete' },
+              { role: 'selectAll' },
+              { type: 'separator' },
+              {
+                label: 'Speech',
+                submenu: [{ role: 'startspeaking' }, { role: 'stopspeaking' }]
+              }
+            ]
+          : [{ role: 'delete' }, { type: 'separator' }, { role: 'selectAll' }])
+      ]
+    },
+    {
+      label: '&View',
+      submenu: [
+        { role: 'reload' },
+        { role: 'forceReload' },
+        { role: 'toggleDevTools' },
+        { type: 'separator' },
+        { role: 'resetZoom' },
+        { role: 'zoomIn' },
+        { role: 'zoomOut' },
+        { type: 'separator' },
+        { role: 'togglefullscreen' }
+      ]
+    },
+    {
+      label: '&Help',
+      submenu: [
+        {
+          label: 'Usage Guide',
+          click: async () => {
+            await shell.openExternal(
+              'https://rotki.readthedocs.io/en/latest/usage_guide.html'
+            );
+          }
+        },
+        {
+          label: 'Frequenty Asked Questions',
+          click: async () => {
+            await shell.openExternal(
+              'https://rotki.readthedocs.io/en/latest/faq.html'
+            );
+          }
+        },
+        { type: 'separator' },
+        {
+          label: 'Release Notes',
+          click: async () => {
+            await shell.openExternal(
+              'https://rotki.readthedocs.io/en/latest/changelog.html'
+            );
+          }
+        },
+        { type: 'separator' },
+        {
+          label: 'Issue / Feature Requests',
+          click: async () => {
+            await shell.openExternal('https://github.com/rotki/rotki/issues');
+          }
+        },
+        {
+          label: 'Logs Directory',
+          click: () => {
+            const logsDir = path.join(app.getPath('userData'), 'rotki', 'logs');
```

Comment:
This for some reason did not work for me in Linux. But very nice idea.

Reply:
Does nothing open at all? Or the wrong folder?

Reply:
Probably variable needs to be adjusted per OS. It does nothing.

Reply:
What is the directory where rotki logs go on Linux?

Reply:
Hah I see now. When ran in developments, so with `electron:serve` we always use the current directory. But that should not matter.

And in the output in the console I see this:
```
xdg-open: file '/home/lefteris/.config/rotki/rotki/logs' does not exist
```

While the proper directory is: `~/.config/rotki/logs`. There is one extra rotki there for Linux. So in Linux it seems `app.getPath('userData')`  returns `~/.config/rotki/` which does indeed make sense. `~/.config/` is too generic.

Does it not do the same in Windows?

Reply:
In windows `app.getPath('userData')` yields `%APPDATA%/rotki`, but logs are in '`%APPDATA%/rotki/rotki/logs`, so there is an extra `rotki` for some reason.
![image](https://user-images.githubusercontent.com/27592/79074518-264e9900-7ced-11ea-8623-74c6de32526c.png)

Here is the content of `%appdata%/rotki`:
![image](https://user-images.githubusercontent.com/27592/79074557-6281f980-7ced-11ea-9875-a1dd73d6f897.png)

Reply:
I'm looking at the logs code in `py-handler.ts` and it seems that there is _another_ log dir as well (at least on Windows), which is `%appdata%/rotki/Electron/logs`. It seems that `app.getPath('logs');` should work regardless of system, but in windows that only gives you the Electron logs without the backend logs.